fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/NotificationService.java

```diff
@@ -163,6 +196,16 @@ private void displayNotifications() throws IOException {
         }
     }
 
+    private void displayMessageKeys() {
+        List<String> keys = findMessages();
+        logger.warn("------------------------------------------------------------------------");
+        logger.warn("Message Keys:");
+        logger.warn("------------------------------------------------------------------------");
+        for(String key: keys) {
+            logger.warn(key);
```

Comment:
S: How would I know what key goes with what message?  Should this not just be an extension of the usual notification printing such that each notification is preceded by its corresponding key?  I think that also streamlines the code a bit.